### PR TITLE
Update dev docs

### LINKED ISSUE #174: Development docs out of date
The docs in https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/development.md still say that you need to make a `tags.json` file, it should be updated with the `"tagTemplates"` feature in `meta.json`.

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary

fixes #174 
## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [x] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -22,18 +22,23 @@ In this folder, there are 4 items:
 ```json
 {
     "IsLinux" : true
+    // tags: This is a JSON array that describes the tags the image should have:
```
Comment (TravisEz13) [inline docs/development.md]: if we want to be able to use comments in json we need replace all the `-powershell` with `-pwsh`

Comment (RDIL) [reply docs/development.md]: I'll move it out

Comment (TravisEz13) [reply docs/development.md]: `- powershell` in Windows PowerShell which did not support JSON comments

Comment (TravisEz13) [reply docs/development.md]: Either way.

**Comment by RDIL** [comment]
@TravisEz13 same here

**Pull request merged**